TCAT/Cells/LargeDetailTableViewCell.swift

```diff
@@ -201,7 +198,7 @@ class LargeDetailTableViewCell: UITableViewCell {
             self.chevron.isHidden = true
         })
 
-        chevron.transform = chevron.transform.rotated(by: CGFloat.pi)
+        chevron.transform = chevron.transform.rotated(by: chevron.transform == CGAffineTransform(rotationAngle: CGFloat.pi) ? -CGFloat.pi : CGFloat.pi)
```

Comment:
can we either split this line into multiple lines or store the angle inside some variable?